Lib/test/test_asyncio/test_unix_events.py

```diff
@@ -417,6 +417,93 @@ def test_create_unix_connection_ssl_timeout_with_plain_sock(self):
             self.loop.run_until_complete(coro)
 
 
+@unittest.skipUnless(hasattr(os, 'sendfile'),
```

Comment:
Can you add a test like this:

```python
await loop.sock_sendall(5 mb of data)
await loop.sock_sendfile(some file)
await loop.sock_sendall(5 mb of data)
```

just to ensure that sendfile and sendall work fine together.